src/components/scenes/SendScene2.tsx

```diff
@@ -553,22 +554,33 @@ const SendComponent = (props: Props) => {
   const renderUniqueIdentifier = () => {
     const spendTarget = spendInfo.spendTargets[0]
     const uniqueIdentifier = spendTarget?.memo ?? spendTarget?.uniqueIdentifier
-    const { uniqueIdentifierInfo } = getSpecialCurrencyInfo(coreWallet.currencyInfo.pluginId)
-
-    if (uniqueIdentifierInfo != null && spendTarget.publicAddress != null) {
-      const { addButtonText, identifierName, keyboardType } = uniqueIdentifierInfo
+    const [memoOption] = memoOptions
```

Comment:
Create a task to refactor SendScene2 to permit multiple memos for each option type